### ISSUE TITLE
one Light is showing up as a switch in HA - other light shows up as a light

### ISSUE THREAD

**Issue by thethereza** [ollo69/ha_tuya_custom#27]
I've been trying to debug this problem, but I've hit a roadblock.  It looks like there may be a custom API developed for HA on the TUYA servers.  If that's the case, then the problem lies in the API server and not HA.  I've posted about the issue here in more detail:

https://community.home-assistant.io/t/tuya-integration-has-a-bug-and-i-need-help-on-how-to-best-debug-it/223219

**Comment by thethereza** [comment]
(i'm also doing my testing on the custom version so i posted here, though the behavior is the same)

**Comment by ChronoStriker1** [comment]
I am also having this issue.  The two lights in question are Merkury Innovations A21 Smart Multicolor LED Bulbs if that helps troubleshoot

**Comment by ollo69** [comment]
> I've been trying to debug this problem, but I've hit a roadblock. It looks like there may be a custom API developed for HA on the TUYA servers. If that's the case, then the problem lies in the API server and not HA. 

That's correct, the problem is in the API. I also have some Bulb that the API recognize as Switch, there are no way to manage these as lights.
